<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogShow</class>
 <widget class="QWidget" name="LogShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1696</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LogShow{
width: 1697px;
height: 755px;
background: #FFFFFF;
border-radius: 10px;
}</string>
  </property>
  <widget class="QWidget" name="widget_switch" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>1696</width>
     <height>104</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 6px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>75</y>
     <width>1694</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    alternate-background-color: #f2f7fb;

    background: #FFFFFF;
    border-radius: 10px 10px 0px 0px;
    border: 1px solid rgba(170, 189, 212, 0.6);

    font-family: PingFang SC;
    font-weight: 500;
    font-size: 22px;
    color: #333333;
    line-height: 35px;
}

QHeaderView {
    font-family: PingFang SC;
    font-weight: 500;
    font-size: 30px;
    color: #333333;
    line-height: 70px;

    border-top-left-radius: 10px;
    border-top-right-radius: 10px;
    border-bottom-left-radius: 10px;
    border-bottom-right-radius: 10px;
}

QHeaderView::section {
    border: none;
}</string>
   </property>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>新建列</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>1560</x>
     <y>10</y>
     <width>120</width>
     <height>62</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 120px;
height: 62px;
background: #19BC90;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FFFFFF;
line-height: 35px;</string>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
